BUILD.md

```diff
@@ -0,0 +1,39 @@
+Environment
```

Comment:
![image](https://user-images.githubusercontent.com/26923638/236665200-1a41cfd7-4fef-43d7-a592-d83677480642.png)
please fix this format. May be `## Environment`